plugins/keycloak-backend/README.md

```diff
@@ -55,93 +55,98 @@ When using client credentials, the access type must be set to `confidential` and
+             ...
+             schedule: # optional; same options as in TaskScheduleDefinition
+               # supports cron, ISO duration, "human duration" as used in code
+               frequency: { minutes: 1 }
+               # supports ISO duration, "human duration" as used in code
+               timeout: { minutes: 1 }
+               initialDelay: { seconds: 15 }
+     ```
+
+     Use the configured scheduler as follows:
+
+     ```diff
+     // packages/backend/src/plugins/catalog.ts
+     + import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+       export default async function createPlugin(
+         env: PluginEnvironment,
+       ): Promise<Router> {
+         const builder = await CatalogBuilder.create(env);
+
+     +   builder.addEntityProvider(
+     +     KeycloakOrgEntityProvider.fromConfig(env.config, {
+     +       id: 'development',
+     +       logger: env.logger,
+     +       scheduler: env.scheduler,
+     +     }),
+     +   )
+       ...
+       }
+     ```
+
+   - Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file as follows:
+
+     ```diff
+     // packages/backend/src/plugins/catalog.ts
+     + import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+       export default async function createPlugin(
+         env: PluginEnvironment,
+       ): Promise<Router> {
+         const builder = await CatalogBuilder.create(env);
+
+     +   builder.addEntityProvider(
+     +     KeycloakOrgEntityProvider.fromConfig(env.config, {
+     +       id: 'development',
+     +       logger: env.logger,
+     +       schedule: env.scheduler.createScheduledTaskRunner({
+     +         frequency: { minutes: 1 },
+     +         timeout: { minutes: 1 },
+     +         initialDelay: { seconds: 15 }
+     +       }),
+     +     }),
+     +   )
+       ...
+       }
+     ```
 
 ## Limitations
 
-If you have selfsigned/corporate certificate issues, the temporary solution (and not recommended) is to set the following environment variable before starting Backstage:
+If you have self-signed/corporate certificate issues, the temporary solution is to set the following environment variable before starting Backstage:
 
 `NODE_TLS_REJECT_UNAUTHORIZED=0`
 
-### Viewing imported users and groups in backstage
+---
 
-After configuring the plugin as described in the previous sections the plugin
-will import users and groups when started and on a regular basis. Once
-the first import is complete you can list the users from the catalog page
-by select `User`:
+**NOTE**
+
+The temporary solution of setting environment variable is not recommended.
+
+---
+
+### View imported users and groups in Backstage
+
+After configuring the plugin as described in the previous sections the plugin imports the users and groups on a regular basis when started. Once the first import is complete, select `User` to list the users from the catalog page:
```

Comment:
```suggestion
After you have successfully configured the plugin, it imports users and groups each time it is started.

After the first import is complete, select `User` to list the users from the catalog page:
```

Reply:
changing it to:

After configuring the plugin successfully, the plugin imports the users and groups each time when started. 

After the first import is complete, you can select `User` to list the users from the catalog page:
![](./images/users.jpg)